plugins/modules/accesspoint_workflow_manager.py

```diff
@@ -3333,6 +3935,94 @@ def reboot_factory_reset_function(self, ap_list, reboot_or_reset):
+            bulk_config (dict): A dict containing the APs mac address and cofig details which need to update APs.
+
+        Returns:
+            dict: A dictionary containing the result of the buld access point update status.
+        """
+        ap_exist, ap_details = self.get_accesspoint_details(bulk_config)
+        self.payload["access_point_details"] = ap_details
+        ap_update_list = []
+        common_config = {}
+        ap_output_list = []
+        if ap_exist and len(ap_details) > 0:
+            for each_ap in ap_details:
+                ap_config_exists, ap_configuration = self.get_accesspoint_config(each_ap["ap_ethernet_mac_address"])
+                self.log("Access point configuration exists: {0}, Current configuration: {1}"
+                         .format(ap_config_exists, self.pprint(ap_configuration)), "INFO")
+                self.want = bulk_config.get("common_fields_to_change")
+                self.want["mac_address"] = each_ap["mac_address"]
+                ap_name = [ap.get('ap_name') for ap in bulk_config.get("ap_identifier")
+                           if (each_ap["mac_address"] == ap.get('mac_address') or
+                               each_ap["hostname"] == ap.get('hostname') or
+                               each_ap["management_ip_address"] == ap.get('management_ip_address'))]
+                self.want["ap_name"] = ap_name[0]
+                ap_output_list.append(ap_name[0])
+                self.log("Access point WANT configuration exists: {0}, Current configuration: {1}"
+                         .format(ap_config_exists, self.pprint(self.want)), "INFO")
+                consolidated_config = self.config_diff(ap_configuration)
+
+                temp_dict = {}
+                if consolidated_config.get(self.keymap["ap_name"]) is not None:
+                    temp_dict[self.keymap["ap_name"]] = consolidated_config.get(self.keymap["ap_name"])
+                    temp_dict["apNameNew"] = consolidated_config["apNameNew"]
+                    temp_dict[self.keymap["mac_address"]] = consolidated_config[self.keymap["mac_address"]]
+                elif consolidated_config.get(self.keymap["mac_address"]) is not None:
+                    temp_dict[self.keymap["mac_address"]] = consolidated_config.get(self.keymap["mac_address"])
+                ap_update_list.append(temp_dict)
+                common_config.update(consolidated_config)
+
+            common_config["bulk_update"] = True
+            common_config["ap_list"] = ap_update_list
+
+            task_response = self.update_ap_configuration(common_config)
+            self.log("Access Point update response: {0} .".format(task_response), "INFO")
+            responses = {}
+            if task_response and isinstance(task_response, dict):
+                resync_retry_count = self.payload.get("dnac_api_task_timeout")
+                resync_retry_interval = self.payload.get("dnac_task_poll_interval")
```

Comment:
self.log("Starting task polling with timeout: {} and interval: {}".format(resync_retry_count, resync_retry_interval), "INFO")